WORKSPACE

```diff
@@ -52,3 +52,11 @@ new_http_archive(
     url = "http://downloads.typesafe.com/scala/2.11.7/scala-2.11.7.tgz",
     build_file = "tools/build_defs/scala/scala.BUILD",
 )
+
+# only used for the scala test rule
```

Comment:
Now there are two rules to include for scala in the WORKSPACE file, please provide a scala_repositories macros instead (see top of the file for the other languages).